.github/workflows/test-docker.yml

```diff
@@ -0,0 +1,96 @@
+name: Test in docker
```

Comment:
BTW, this commit ('bugfix: missing files from installed deb packages') fixes #44. I would add `Fixes #44` into the commit and into the pull request description.

Reply:
Fixed.